src/components/common/Extensions/UserNavigation/partials/UserSubmenu/consts.ts

```diff
@@ -1,64 +1,129 @@
+import { defineMessages } from 'react-intl';
+
+import {
+  COLONY_DEV_DOCS,
+  COLONY_DISCORD,
+  COLONY_DOCS,
+  COLONY_GITHUB,
+  COLONY_TWITTER,
+  FEATURES_BUGS,
+  WHATS_NEW,
+  PRIVACY_POLICY,
+  TERMS_AND_CONDITIONS,
+} from '~constants';
 import { UserSubmenuItems } from './types';
+import { useFeaturesBugs, useWhatsNew } from '~hooks/useBeamer';
+
+export const menuMessages = defineMessages({
+  getHelp: {
+    id: 'UserSubmenu.getHelp',
+    defaultMessage: 'Get help',
+  },
+  whatsNew: {
+    id: 'UserSubmenu.whatsNew',
+    defaultMessage: "What's new",
+  },
+  featureBugs: {
+    id: 'UserSubmenu.featureBugs',
+    defaultMessage: 'Features & Bugs',
+  },
+  discord: {
+    id: 'UserSubmenu.discord',
+    defaultMessage: 'Discord',
+  },
+  twitter: {
+    id: 'UserSubmenu.twitter',
+    defaultMessage: 'Twitter',
+  },
+  developerDocs: {
+    id: 'UserSubmenu.developerDocs',
+    defaultMessage: 'Developer docs',
+  },
+  github: {
+    id: 'UserSubmenu.github',
+    defaultMessage: 'Github',
+  },
+  privacyPolicy: {
+    id: 'UserSubmenu.privacyPolicy',
+    defaultMessage: 'Privacy policy',
+  },
+  termsOfUse: {
+    id: 'UserSubmenu.termsOfUse',
+    defaultMessage: 'Terms of use',
+  },
+});
 
 export const userSubmenuItems: UserSubmenuItems = {
   'userMenu.contactAndSupportTitle': [
     {
       id: 'getHelp',
-      label: 'Get help',
-      url: '/get-help',
+      label: menuMessages.getHelp,
+      url: COLONY_DOCS,
       icon: 'lifebuoy',
+      external: true,
     },
     {
       id: 'featureRequest',
-      label: 'Feature request',
-      url: '/feature-request',
+      label: menuMessages.whatsNew,
+      url: WHATS_NEW,
       icon: 'star',
+      onClick: useWhatsNew,
```

Comment:
maybe we can not name it as a hook a la `useWhatsNew`, name it `openWhatsNew` and just move it to a helper? Since I see that just calls the Beamer instance, maybe a `utils/beamer` file wouldn't be a bad place for this

Reply:
Adding here discussion from Discord. Basically, I want to contain this implementation as much as possible, so, I will rename but keep in the same location.